native/src/feature/onboarding/component/CodeInput.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import { Text } from "react-native";
+import {
+  CodeField,
+  Cursor,
+  useBlurOnFulfill,
+  useClearByFocusCell,
+} from "react-native-confirmation-code-field";
+import styles from "./styles";
+
+const CELL_COUNT = 6;
+type Props = {
+  onChangeText: (text: string) => void | undefined;
+};
+const CodeInput: React.FC<Props> = ({ onChangeText }) => {
+  const [value, setValue] = useState("");
+  const ref = useBlurOnFulfill({ value, cellCount: CELL_COUNT });
+  const [props, getCellOnLayoutHandler] = useClearByFocusCell({
+    value,
+    setValue,
+  });
+
+  const onSetVerifcationCode = (code: string) => {
+    onChangeText(code);
+    setValue(code);
+  };
+
+  return (
+    <CodeField
+      ref={ref}
+      {...props}
+      value={value}
```

Comment:
```suggestion
      {...props}
      ref={ref}
      value={value}
```

Procura poner `props` al principio.